Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM fedora:25
+
+# Install heketi-cli
+RUN ["/usr/bin/dnf","install","heketi-client","unzip","python","-y"]
+
+# Install kubectl
+ADD https://storage.googleapis.com/kubernetes-release/release/v1.6.1/bin/linux/amd64/kubectl /usr/local/bin/
+
+# Make kubectl executable
+RUN ["chmod","+x","/usr/local/bin/kubectl"]
```

Comment:
Can you putt the `ADD` at the beginning of the Dockerfile and combine `RUN` e.g. https://github.com/docker-library/golang/blob/master/1.8/alpine/Dockerfile#L12
This reduce the size of the Dockerfile